README.md

```diff
@@ -0,0 +1,3 @@
+## Hello World
+
+Its a real crappy day out there.
```

Comment:
Should this be "it's"? This is a contraction.

Reply:
Great catch, updated to 'it's'